src/KNXClient.ts

```diff
@@ -110,7 +110,11 @@ export type KNXClientOptions = {
 	KNXQueueSendIntervalMilliseconds?: number
 	/** Enables sniffing mode to monitor KNX */
 	sniffingMode?: boolean
-} & KNXLoggerOptions
+	/** Log level */
+	loglevel?: LogLevel
+	/** Set Prefix for logs */
+	setPrefix?: string
+}
```

Comment:
I would use KNXLoggerOptions here, mention that you removed `debug` flag in PR. This is a breaking change